src/components/calcite-input/calcite-input.tsx

```diff
@@ -495,6 +500,10 @@ export class CalciteInput {
     this.setValue(this.defaultValue, nativeEvent);
   };
 
+  private sanitizeNumberString(value: string): string {
```

Comment:
Should this go into a utility file somewhere? It seems like a useful utility.

Reply:
Just added it :)